docs/Tutorial.md

```diff
@@ -51,64 +50,67 @@ ONE-vscode has 6 screen components.
+        - Under each *.cfg file, any files generated by running `onecc` with the *.cfg will be shown. For example, `*.circle` file and files after backend codegen.
 3. `Toolchain view`
-    - This shows installed toolchain.
+    - This shows a tree view of installed toolchains.
     - You can also install a new toolchain or uninstall an existing toolchain.
 4. `Target Device view`
-    - This shows tree view of devices or simulators that can run for inference.
+    - This shows a tree view of devices or simulators that can run for inference.
 5. `ONE cfg editor`
-    - This provides easier way to edit ONE cfg file.
+    - This provides easier way to edit a ONE cfg file.
 6. `ONE-vscode Log`
     - This shows logs. You can check detailed reason of error or warning.
 
-<!-- Rearrange content below -->
-
 ## Workspace
 
 You need to open a folder (`File > Open Folder` menu) to start with ONE-vscode. After opening a folder, a [_workspace_ is constructed by vscode.](https://code.visualstudio.com/docs/editor/workspaces).
 
 The folder for ONE-vscode needs to have the following files:
 - Your model files
-  - Tensorflow model file(*.pb, saved model or Keras model)
-  - Tensorflow Lite model (*.tflite)
-  - Onnx model (*.onnx)
+  - TensorFlow model file(*.pb, saved model or Keras model)
+  - TensorFlow Lite model (*.tflite)
+  - ONNX model (*.onnx)
 - ONE *.cfg files (You can create this file later)
 
-## Install the toolchain
+## Install a toolchain
 
-You need the ONE toolchain. Let's install the ONE toolchain. (You should install the in-house backend to follow this.)
+A toolchain contains [ONE command tools](https://github.com/Samsung/ONE/tree/master/compiler/one-cmds) and other backend-specific tools. Developers or vendors of backend can distribute backend-specific toolchain and put installation feature inside their backend extension.
 
-<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=600 />
+To install a backend-specific toolchain, you first need to install the backend extension. Then, click `+` icon on the Toolchain view. You can also easily remove installed toolchain by clicking `X` icon in the Toolchain view.
 
 ## Set default toolchain
 
-To compile model using ONE toolchain, you must specify the default toolchain. Default toolchain refers to which toolchain you will use to run ONE related tools. To specify a default toolchain, if you mouse up the toolchain item you want to use in Toolchain view, :heavy_check_mark: icon appears. Click this icon to specify it as default toolchain. If toolchain is specified as default toolchain, the icon changes.
+You can install multiple instances of toolchains. For example, some backend supports multiple versions of toolchain to be installed. Or you might installed more than one backend extensions installed and installed one or more toolchain per backend extension.
+
```

Comment:
The purpose of writing this tutorial was to tell user that default toolchain setting is essential to execute config. Could you add the contents about this?
> To compile model using ONE toolchain, you must specify the default toolchain.

Reply:
I think current statement at L88

> To decide which toolchain will be used by default, you need to specify the default toolchain.

is same with the statement you'd like to add. 

Do you want to replace current statement with the one you wrote like the following?

`To decide which toolchain will be used by default, you need to specify the default toolchain.`
to 
`To compile model using ONE toolchain, you must specify the default toolchain.`
?

Reply:
`To execute config file in ONE explorer, you must specify the default toolchain.`
How about it?